cmd/libs/go2idl/args/args.go

```diff
@@ -61,6 +61,9 @@ type GeneratorArgs struct {
 
 	// If true, only verify, don't write anything.
 	VerifyOnly bool
+
```

Comment:
This invalidates the comment above for GeneratorArgs. How about changing it to "GeneratorArgs has arguments that needs to be passed to generators"?